File: Back/src/Fullstack.Domain/Evento.cs
namespace Fullstack.Domain;
public class Evento
{
    public int Id { get; set; }
    public string Local { get; set; }
    public DateTime? DataEvento { get; set; }
    public string Tema { get; set; }
    public int QtdPessoas { get; set; }
    public string ImagemUrl { get; set; }
    public int Telefone { get; set; }
    public string Email { get; set; } 
    public IEnumerable<Lote> Lotes { get; set; }
    public IEnumerable<RedeSocial> RedeSociais { get; set; }
    public IEnumerable<Palestrante> Palestrantes { get; set; }
    public IEnumerable<PalestranteEvento> PalestrantesEventos { get; set; }
}

File: Front/FullStack-App/src/app/models/Evento.ts
import { RedeSocial } from './RedeSocial';
import { Lote } from "./Lote";
import { Palestrante } from './Palestrante';

export interface Evento {
  id: number;
  local: string;
  dataEvento?: Date;
  tema: string;
  qtdPessoas: number;
  imagemUrl: string;
  telefone: string;
  email: string;
  userId: number;
  lotes: Lote[];
  redesSociais: RedeSocial[];
  palestrantesEventos: Palestrante[];
}
